systemtest/src/test/java/io/strimzi/systemtest/MultipleNamespaceST.java

```diff
@@ -7,112 +7,68 @@
      */
     @Test
     @Tag(REGRESSION)
-    void testKafkaInDifferentNsThanClusterOperator() throws InterruptedException {
-        String kafkaName = kafkaClusterName(CLUSTER_NAME + "-second");
-
-        secondNamespaceResources.kafkaEphemeral(CLUSTER_NAME + "-second", 3).done();
-
-        LOGGER.info("Waiting for creation {} in namespace {}", kafkaName, TO_NAMESPACE);
-        kubeClient.namespace(TO_NAMESPACE);
-        kubeClient.waitForStatefulSet(kafkaName, 3);
+    void testKafkaInDifferentNsThanClusterOperator() {
+        LOGGER.info("Deploying Kafka in different namespace than CO when CO watches multiple namespaces");
+        checkKafkaInDiffNamespaceThanCO();
     }
 
     /**
      * Test the case when MirrorMaker will be deployed in different namespace across multiple namespaces
      */
     @Test
     @Tag(REGRESSION)
-    void testDeployMirrorMakerAcrossMultipleNamespace() throws InterruptedException {
-        String kafkaName = CLUSTER_NAME + "-target";
-        String kafkaSourceName = kafkaClusterName(CLUSTER_NAME);
-        String kafkaTargetName = kafkaClusterName(kafkaName);
-
-        secondNamespaceResources.kafkaEphemeral(kafkaName, 3).done();
-        secondNamespaceResources.kafkaMirrorMaker(CLUSTER_NAME, kafkaSourceName, kafkaTargetName, "my-group", 1, false).done();
-
-        LOGGER.info("Waiting for creation {} in namespace {}", CLUSTER_NAME + "-mirror-maker", TO_NAMESPACE);
-        kubeClient.namespace(TO_NAMESPACE);
-        kubeClient.waitForDeployment(CLUSTER_NAME + "-mirror-maker", 1);
-    }
-
-    @BeforeEach
-    void createSecondNamespaceResources() {
-        kubeClient.namespace(TO_NAMESPACE);
-        secondNamespaceResources = new Resources(namespacedClient());
-        kubeClient.namespace(CO_NAMESPACE);
-    }
-
-    @AfterEach
-    void deleteSecondNamespaceResources() throws Exception {
-        secondNamespaceResources.deleteResources();
-        waitForDeletion(TEARDOWN_GLOBAL_WAIT, TO_NAMESPACE, CO_NAMESPACE, TO_NAMESPACE);
-        kubeClient.namespace(CO_NAMESPACE);
+    void testDeployMirrorMakerAcrossMultipleNamespace() {
+        LOGGER.info("Deploying Kafka MirrorMaker in different namespace than CO when CO watches multiple namespaces");
+        checkMirrorMakerForKafkaInDifNamespaceThanCO();
     }
 
     @BeforeAll
     static void createClassResources(TestInfo testInfo) {
         LOGGER.info("Creating resources before the test class");
-        applyRoleBindings(CO_NAMESPACE, CO_NAMESPACE, TO_NAMESPACE);
-        // 050-Deployment
-        testClassResources.clusterOperator(String.join(",", CO_NAMESPACE, TO_NAMESPACE)).done();
+        applyRoleBindings(CO_NAMESPACE, CO_NAMESPACE, SECOND_NAMESPACE);
+
+        LOGGER.info("Deploying CO to watch multiple namespaces");
+        testClassResources.clusterOperator(String.join(",", CO_NAMESPACE, SECOND_NAMESPACE)).done();
 
         classResources = new Resources(namespacedClient());
```

Comment:
No need to create second class resources. You can create kafka via `testClassResources`

Reply:
Okay, you are code owner here, maybe you know it better

Reply:
Done